### DISCUSSION TITLE
API not working  error through "NoRecipients"

### DISCUSSION THREAD

**Discussion by rccess** [postalserver/postal#2245] [Help with using Postal] [answered]
can you please advise on it

{
"to": [
"[xxxxxxxxxxxxx@gmail.com](mailto:xxxxxxxxxxxxx@gmail.com)"
],
"from": "[xxxxxxxxxxx@xxxxxx.lt](mailto:xxxxxxxxxxx@xxxxxx.lt)",
"subject": "Email API Test",
"plain_body": "hello this email api checking"
}

![image](https://user-images.githubusercontent.com/32220165/203775912-d809b93a-d7e9-4f6a-92e5-29d44766d623.png)

{
    "status": "error",
    "time": 0.0,
    "flags": {},
    "data": {
        "code": "NoRecipients",
        "message": "There are no recipients defined to receive this message"
    }
}

**Comment by willpower232** [comment, marked as answer]
I think you need to change the `Text` dropdown to `JSON`.

You might need to put the `]` on the previous line as well, I'm not sure how flexible the JSON handling is.
